### PR TITLE
Add error message to empty catch blocks in aad-register.ps1. 

### PR DESCRIPTION
**Pull request by WilliamBerryiii**
There are empty catch blocks that result in script failure, without reporting where the failure occurred, when falling through user credential collection in app registration. 
This closes issue #1045.

### CODE REVIEW

File: deploy/scripts/aad-register.ps1 [hunk 1]
```diff
@@ -224,6 +224,7 @@ Function New-ADApplications() {
                     -TenantId $tenantId `
                 }
             catch {
+                Write-Host "An exception was encountered collecting user credentials when registering $($applicationDisplayName): $($_.Exception.Message)"
```
Comment (luiscantero) [inline deploy/scripts/aad-register.ps1:227]: Let's avoid passive voice when possible and simplify, e.g.: "Failed collecting user credentials while registering $($applicationDisplayName): $($_.Exception.Message)"

File: deploy/scripts/aad-register.ps1 [hunk 2]
```diff
@@ -246,6 +247,7 @@ Function New-ADApplications() {
                     $user = (Get-AzureADUser -SearchString $creds.Account.Id)[0]
                 }
                 catch {
+                    Write-Host "Getting user principal for $($creds.Account.Id) failed when searching by the acount ID."
```
Comment (luiscantero) [inline deploy/scripts/aad-register.ps1:250]: Fix typo "acount" and avoid passive voice, e.g. "Failed getting user principal for $($creds.Account.Id) while searching by account ID"

**Review by luiscantero — CHANGES_REQUESTED** [review]

File: deploy/scripts/aad-register.ps1 (hunk 1 above)
Comment (WilliamBerryiii) [reply deploy/scripts/aad-register.ps1:227]: Thanks and great callout!  Will update.

File: deploy/scripts/aad-register.ps1 (hunk 2 above)
Comment (WilliamBerryiii) [reply deploy/scripts/aad-register.ps1:250]: Thanks and great callout!  Will update.

**Comment by WilliamBerryiii** [comment]
Addressed code review comments, updated branch with latest change sets and pushed. Thanks for the review!

**Review by luiscantero — APPROVED** [review]

**Pull request merged**